packages/11ty/content/_data/eleventyComputed.js

```diff
@@ -25,20 +25,39 @@ module.exports = {
     })
   },
   eleventyNavigation: {
+    /**
+     * Explicitly define page data properties used in the TOC
+     * since eleventyNavigation does not include the entire page object
+     */
+    data: (data) => {
+      return {
+        abstract: data.abstract,
+        contributor: data.contributor,
+        figure: data.figure,
+        image: data.image,
+        label: data.label,
+        layout: data.layout,
+        object: data.object,
+        online: data.online,
```

Comment:
Had we discussed renaming `online` to something else?

Reply:
We did but we didn't come up with anything, I'm find with leaving it for now